iam/api-client/src/main/java/com/google/iam/snippets/ServiceAccountKeys.java

```diff
@@ -15,47 +15,54 @@
 
 package com.google.iam.snippets;
 
+// [START iam_create_key]
+// [START iam_list_keys]
+// [START iam_delete_key]
 import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
 import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
 import com.google.api.client.json.jackson2.JacksonFactory;
 import com.google.api.services.iam.v1.Iam;
 import com.google.api.services.iam.v1.IamScopes;
-import com.google.api.services.iam.v1.model.CreateServiceAccountKeyRequest;
 import com.google.api.services.iam.v1.model.ServiceAccountKey;
 import java.io.IOException;
 import java.util.Collections;
+// [END iam_create_key]
+// [END iam_list_keys]
+// [END iam_delete_key]
+
+// [START iam_create_key]
+import com.google.api.services.iam.v1.model.CreateServiceAccountKeyRequest;
+// [END iam_create_key]
+
+// [START iam_list_keys]
 import java.util.List;
+// [END iam_list_keys]
+
+// [START iam_create_key]
+// [START iam_list_keys]
+// [START iam_delete_key]
 
 public class ServiceAccountKeys {
 
   private final Iam service;
 
   public ServiceAccountKeys() throws Exception {
-    GoogleCredential credential =
-        GoogleCredential.getApplicationDefault()
-            .createScoped(Collections.singleton(IamScopes.CLOUD_PLATFORM));
-
-    service =
-        new Iam.Builder(
-                GoogleNetHttpTransport.newTrustedTransport(),
-                JacksonFactory.getDefaultInstance(),
-                credential)
-            .setApplicationName("service-account-keys")
-            .build();
+    GoogleCredential credential = GoogleCredential.getApplicationDefault()
+        .createScoped(Collections.singleton(IamScopes.CLOUD_PLATFORM));
+
+    service = new Iam.Builder(GoogleNetHttpTransport.newTrustedTransport(), JacksonFactory.getDefaultInstance(),
+        credential).setApplicationName("service-account-keys").build();
   }
 
+// [END iam_create_key]
+// [END iam_list_keys]
+// [END iam_delete_key]
+
   // [START iam_create_key]
   public ServiceAccountKey createKey(String serviceAccountEmail) throws IOException {
 
-    ServiceAccountKey key =
-        service
-            .projects()
-            .serviceAccounts()
-            .keys()
-            .create(
-                "projects/-/serviceAccounts/" + serviceAccountEmail,
-                new CreateServiceAccountKeyRequest())
-            .execute();
+    ServiceAccountKey key = service.projects().serviceAccounts().keys()
+        .create("projects/-/serviceAccounts/" + serviceAccountEmail, new CreateServiceAccountKeyRequest()).execute();
```

Comment:
You might wish to ping the author, they were trying to illustrate something w/ these lines, you've made it a bit harder to see that.